guides/source/active_record_querying.md

```diff
@@ -25,9 +25,13 @@ If you're used to using raw SQL to find database records, then you will generall
 
```

Comment:
The DETAIL section in the PR description gives a good idea of what has changed, but also what will be tackled as follow ups.